pages/upgrade-guide/3.5.md

```diff
@@ -38,7 +40,7 @@ would create the following structure:
 ```
 because `<div/>` was replaced with `<div></div>` & `<span/>` with `<span></span>`.
 
-jQuery 3.5.0 has changed `jQuery.htmlPrefilter` to be an identity function. That means that the above `jQuery` call would now create above HTML structure only in XML mode of HTML (called also as XHTML) but in regular HTML mode you would now get:
+jQuery 3.5.0 has changed `jQuery.htmlPrefilter` to be an identity function. In HTML mode auto-closing tags are being replaced with their opening tags, making `<div/><span/>` interpreted as `<div><span>`; the browser will automatically close these tags at the end of input. That means that the above `jQuery` call would now create above HTML structure only in XML mode of HTML (called also as XHTML) but in regular HTML mode you would now get:
```

Comment:
```suggestion
In jQuery 3.5.0, the `jQuery.htmlPrefilter` method always returns its argument unchanged. Modern browsers usually parse and render HTML strings in HTML5 mode. Elements such as `<p>` and `<span>` are never self-closing in HTML5, so the browser interprets a string like `<p/><span/>` as `<p><span>` and leaves the tags open. The browser closes the tags at the end of the string or at an element that cannot be contained in the element, so another `<p>` tag closes an existing open `<p>` tag.
```

Reply:
That sounds good, maybe I'd just change HTML5 to HTML as that's the official naming now:
```suggestion
In jQuery 3.5.0, the `jQuery.htmlPrefilter` method always returns its argument unchanged. Modern browsers usually parse and render HTML strings in HTML mode. Elements such as `<p>` and `<span>` are never self-closing in HTML, so the browser interprets a string like `<p/><span/>` as `<p><span>` and leaves the tags open. The browser closes the tags at the end of the string or at an element that cannot be contained in the element, so another `<p>` tag closes an existing open `<p>` tag.
```